Add changelog entries for Core/Infra (#3898)
elastic/docs-content: release-notes/elastic-cloud-serverless/index.md

* Introduces a new endpoint scheme for SIEM migration [#219597]({{kib-pull}}219597)
* Extends default log pattern on server side to include error information [#219940]({{kib-pull}}219940)

> * Ensures config reload on ..data symlink switch for `CSI` driver support [#127628](https://github.com/elastic/elasticsearch/pull/127628)

Reviewer: ```suggestion
* Ensures config reload on `..data` symlink switch for CSI driver support [#127628](https://github.com/elastic/elasticsearch/pull/127628)
```

* Adds an **ES|QL** control option to the dashboard controls dropdown [#219495]({{kib-pull}}219495)
* Enables full-text search in `STATS ... WHERE` **ES|QL** queries [#220691]({{kib-pull}}220691)
* Prevents downloading trained models that are already present in other spaces and displays a warning in Machine Learning [#220238]({{kib-pull}}220238)

> * Creates and deletes per-project `objectstore`

Reviewer: ```suggestion
* Adds support for creating and deleting per-project object stores
```

* Fixes incorrect validation for names containing asterisks in **ES|QL** [#219832]({{kib-pull}}219832)
* Fixes overridden SSL config in full agent policy advanced YAML for Fleet [#219902]({{kib-pull}}219902)

> * Avoids nested docs in `painless execute api` [#127991](https://github.com/elastic/elasticsearch/pull/127991)

Reviewer: ```suggestion
* Avoids nested docs in Painless execute API [#127991](https://github.com/elastic/elasticsearch/pull/127991)
```

* Filters inference connectors that lack existing endpoints in **Connectors** [#217641]({{kib-pull}}217641)
* Fixes focusability and keyboard access issues with the **Export** tab in the **Share this dashboard** modal [#217313]({{kib-pull}}217313)

* Improves handling of empty response [#125562](https://github.com/elastic/elasticsearch/pull/125562)
* Adds leniency to missing array values in mustache [#126550](https://github.com/elastic/elasticsearch/pull/126550)

> * Adds a custom toString to `DynamicMap` [#126562](https://github.com/elastic/elasticsearch/pull/126562)

Reviewer: ```suggestion
* Adds a custom `toString` to `DynamicMap` [#126562](https://github.com/elastic/elasticsearch/pull/126562)
```

* Updates data feeds for anomaly detection jobs to exclude Elastic Agent and Beats processes [#213927]({{kib-pull}}213927)
* Adds Mustache lambdas for alerting action [#213859]({{kib-pull}}213859)
* Adds 'page reload' screen reader warning [#214822]({{kib-pull}}214822)

> * Supports `reserved state` to be spread across several files

Reviewer: ```suggestion
* Adds support for spreading a single reserved state across several files instead of a single file to enable projects to be created in one cluster state update instead of several.
```
